### PR TITLE
feat(avatar): SmallVideo uses the existing Avatar component.

### PR DESCRIPTION
**Pull request by virtuacoplenny**
Related torture tests: https://github.com/jitsi/jitsi-meet-torture/pull/88

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
Jenkins, test this please.

File: modules/UI/videolayout/SmallVideo.js [hunk 1]
```diff
@@ -652,21 +655,42 @@ SmallVideo.prototype.updateView = function () {
...
+ * url.
+ *
+ * @param {string} avatarUrl - The uri to the avatar image.
+ * @returns {void}
+ */
 SmallVideo.prototype.avatarChanged = function (avatarUrl) {
-    var thumbnail = $('#' + this.videoSpanId);
-    var avatarSel = this.$avatar();
+    const $thumbnail = this.$avatar();
+    const thumbnail = $thumbnail && $thumbnail.get(0);
```
Comment (virtuacoplenny) [inline modules/UI/videolayout/SmallVideo.js]: They're different but end in the same result. Assuming this.$avatar() is always a jquery lookup, $thumbnail will always be defined but may not have any elements in its internal array of found elements. If it has no elements, the get(0) call will return undefined anyway. This makes me think the truthy check for $thumbnail isn't even needed so I'll get rid of it.

File: modules/UI/videolayout/SmallVideo.js [hunk 2]
```diff
@@ -652,21 +655,42 @@ SmallVideo.prototype.updateView = function () {
...
+
+/**
+ * Unmounts any attached react components (particular the avatar image) from
+ * the avatar container.
+ *
+ * @returns {void}
+ */
+SmallVideo.prototype.removeAvatar = function () {
+    const $thumbnail = this.$avatar();
+    const thumbnail = $thumbnail && $thumbnail.get(0);
```
Comment (paweldomas) [inline modules/UI/videolayout/SmallVideo.js]: there's another one right here

File: modules/UI/videolayout/SmallVideo.js [hunk 3]
```diff
@@ -664,7 +664,7 @@ SmallVideo.prototype.updateView = function () {
  */
 SmallVideo.prototype.avatarChanged = function (avatarUrl) {
     const $thumbnail = this.$avatar();
-    const thumbnail = $thumbnail && $thumbnail.get(0);
+    const thumbnail = $thumbnail.get(0);
```
Comment (paweldomas) [inline modules/UI/videolayout/SmallVideo.js]: I guess $thumbnail var is no longer needed

**Pull request merged**